pages/common/nc.md

```diff
@@ -1,6 +1,6 @@
 # nc
 
-> A versatile utility for redirecting IO into a network stream.
+> Redirect IO into a network stream in a versatile way.
```

Comment:
Hmm. I think the original is better.

Reply:
It's because "Redirect [...] in a versatile way" isn't really equivalent to "versatile tool", right? Yeah, I think agree on that.